discord_slash/utils/manage_commands.py

```diff
@@ -235,3 +313,65 @@ def create_choice(value: str, name: str):
         "value": value,
         "name": name
     }
+
+
+def create_permission(id:int, id_type: int, permission: typing.Union[bool, SlashCommandPermissionsType]):
+    """
+    Create a single command permission.
+
+    :param id: Target id to apply the permission on.
+    :param id_type: Type of the id, :class:`..model.SlashCommandPermissionsType`.
+    :param permission: State of the permission. ``True`` to allow access, ``False`` to disallow access.
+    :return: dict
+
+    .. note::
+        For @everyone permission, set id_type as role and id as guild id.
+    """
+    if not isinstance(id_type, int) or isinstance(id_type, bool): #Bool values are a subclass of int
+        original_type = id_type
+        id_type = SlashCommandPermissionsType.from_type(original_type)
+        if id_type is None:
+            raise IncorrectType(f"The type {original_type} is not recognized as a type that Discord accepts for slash command permissions.")
+    return {
+        "id": id,
+        "type": id_type,
+        "permission": permission
+    }
+
+
+def create_multi_ids_permission(ids: typing.List[int], id_type: int, permission: bool):
+    """
+    Creates a list of permissions from list of ids with common id_type and permission state.
+
+    :param id: List of target ids to apply the permission on.
+    :param id_type: Type of the id. 
+    :param permission: State of the permission. ``True`` to allow access, ``False to disallow access.
```

Comment:
```suggestion
    :param permission: State of the permission. ``True`` to allow access, ``False`` to disallow access.
```
Sphinx was complaining :(
````
C:\Users\___\Documents\Coding\test py-slash\discord-py-slash-command\discord_slash\utils\manage_commands.py:docstring of discord_slash.utils.manage_commands.create_multi_ids_permission:5: WARNING: Inline literal start-string without end-string.
```

Reply:
what didn't I commit your suggested change previously lol